client/spa/js/learning-resource/learning-resource.controller.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+
+var Backbone = require('../vendor/index').Backbone;
+var $ = require('../vendor/index').$;
+var Model = require('./learning-resource.model');
+var View = require('./learning-resource.view');
+
+module.exports = Backbone.Controller.extend({
+
+  routes: {
+    'learning-resource/:': 'list',
+    'learning-resource/:id': 'showLearningResource'
+  },
+
+  initialize: function(){
+    this.options.container = this.options.container || 'body';
+    this.model = new Model();
+    this.view = new View({model: this.model});
```

Comment:
I'm not sure these initializations are necessary.  `this.view` is set in `showLearningResource` and `this.model` could be set in `fetchModel`, which seems like it might be preferable to updating the `id` on an already existing model.